src/sessions/video-session-handler.ts

```diff
@@ -79,8 +79,13 @@ export default class VideoSessionHandler extends BaseSessionHandler {
     };
   }
 
-  handleConversationUpdate (session: IExtendedMediaSession, conversationUpdate: ConversationUpdate): void {
-    super.handleConversationUpdate(session, conversationUpdate);
+  handleConversationUpdate (update: ConversationUpdate, sessions: IExtendedMediaSession[]) {
+    sessions
```

Comment:
Little heads up that the session-manager will only be passing in video sessions: `sessions.filter(s => s.sessionType === handler.sessionType)`